commit cab235b8ecc023b41272b2ceb5e592c2b45bec89
Author: Dave Stevenson <dave.stevenson@raspberrypi.org>
Date:   Fri Sep 1 17:38:27 2017 +0100

    vcsm: Fix "struct vm_operations_struct should normally be const"
    
    Signed-off-by: Dave Stevenson <dave.stevenson@raspberrypi.org>

diff --git a/drivers/char/broadcom/vc_sm/vmcs_sm.c b/drivers/char/broadcom/vc_sm/vmcs_sm.c
index 01da0badd8af..4a1b22fe2ced 100644
--- a/drivers/char/broadcom/vc_sm/vmcs_sm.c
+++ b/drivers/char/broadcom/vc_sm/vmcs_sm.c
@@ -1248,7 +1248,7 @@ static int vcsm_vma_fault(struct vm_area_struct *vma, struct vm_fault *vmf)
 	}
 }
 
-static struct vm_operations_struct vcsm_vm_ops = {
+static const struct vm_operations_struct vcsm_vm_ops = {
 	.open = vcsm_vma_open,
 	.close = vcsm_vma_close,
 	.fault = vcsm_vma_fault,
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1679_cab235b8ecc023b41272b2ceb5e592c2b45bec89.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1679_cab235b8ecc023b41272b2ceb5e592c2b45bec89.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
